pkg/cli/build.go

```diff
@@ -207,8 +207,7 @@ func addLocalImage(cmd *cobra.Command) {
 }
 
 func addConfigFlag(cmd *cobra.Command) {
-	const configFlag = "f"
-	cmd.Flags().StringVar(&configFilename, configFlag, "cog.yaml", "The name of the config file.")
+	cmd.Flags().StringVarP(&configFilename, "file", "f", "cog.yaml", "The name of the config file.")
```

Comment:
`-f/--file` instead of `--f` to be more consistent with Docker. Breaking is fine since it didn't work before anyway.